components/SingleReport.vue

```diff
@@ -29,6 +30,14 @@ const createLabelsFromCriteria = (currentCriterion) => {
   const criterion = props.criteria[currentCriterion]?.name || currentCriterion;
   return [criterion, criterion];
 };
+
+const getMarkdownContent = (content) => {
+  try {
+    return marked.parse(content);
+  } catch {
+    return content;
+  }
+};
```

Comment:
If for some reason we get invalid markdown we will always revert to the content passed so the user does not have an empty page